protocol-jerky/src/main/scala/org/ensime/jerk/JerkFormats.scala

```diff
@@ -19,7 +19,7 @@ private object JerkConversions extends DefaultJsonProtocol with FamilyFormats {
   implicit override def eitherFormat[A: JsonFormat, B: JsonFormat]: JsonFormat[Either[A, B]] = super.eitherFormat[A, B]
   // Note that its not possible to override an object in scala, so we
   // just define a new one that wins the race.
-  implicit val symbolFormat = SymbolJsonFormat
+  implicit val symbolFormat: SymbolJsonFormat.type = SymbolJsonFormat
```

Comment:
Weird bizarre. I understand why it's needed though but I wouldn't have realised it before you added it.